src/main/java/com/gtnewhorizons/modularui/api/math/MathExpression.java

```diff
@@ -227,7 +283,8 @@ public enum Operator {
         MULTIPLY("*"),
         DIVIDE("/"),
         MOD("%"),
-        POWER("^");
+        POWER("^"),
+        EXPONENT("e");
```

Comment:
A better name for this would be scientific notation, or just `SCIENTIFIC` in this case

Reply:
👍